src/components/facets/FacetList.vue

```diff
@@ -6,7 +6,7 @@
     @scroll="onScroll">
 
     <div class="hawk-facet-rail__heading" @click="toggleFacetMobileMenu">
-      {{ $t("Narrow Results") }}
+      {{ $t("Filter By") }}
```

Comment:
I think the branch needs to be rebased or something. Isn't this update from a previous task?

Reply:
This is an update form a previous task.